Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -318,6 +318,64 @@ CommandLineArgs::CommandLineArgs(const std::vector<std::wstring>& args)
             throw hresult_invalid_argument(msg.c_str());
         }
     }
+
+    // Set up output directories
+    {
+        std::filesystem::path PerfOutputPath(sPerfOutputPath);
```

Comment:
Do sPerfOutputPath, sBaseOutputPath, and sPerIterationDataPath need to be initialized? Does the std::filesystem::path constructor handle unintialized wstrings?

Reply:
Default string constructor sets the string to null string (L""), which I verified in the debugger.  Since they're empty, std::filesystem::path handles the case without a problem.

I guess one thing to note is that the paths are initialized to default values, even if the outputs are not turned on... I err'd on the side of trying to keep the code simpler, and to have the paths always initialized to something, rather than only init them if -perfoutput or -periterationperf or -savetensor is specified.  And it's one more place where future additions would need to be added for more output options...

Reply:
Thanks for verifying! Sounds good to me